lib/sisimai/bite/email/exim.rb

```diff
@@ -0,0 +1,522 @@
+           all[ ](?:
+             host[ ]address[ ]lookups[ ]failed[ ]permanently
+            |relevant[ ]MX[ ]records[ ]point[ ]to[ ]non[-]existent[ ]hosts
+            )
+          |Unrouteable[ ]address
+          )
+        }x,
+        # transports/appendfile.c:2567|  addr->user_message = US"mailbox is full";
+        # transports/appendfile.c:3049|  addr->message = string_sprintf("mailbox is full "
+        # transports/appendfile.c:3050|  "(quota exceeded while writing to file %s)", filename);
+        mailboxfull: %r/(?:mailbox[ ]is[ ]full:?|error:[ ]quota[ ]exceed)/x,
+        # routers/dnslookup.c:328|  addr->message = US"an MX or SRV record indicated no SMTP service";
+        # transports/smtp.c:3502|  addr->message = US"no host found for existing SMTP connection";
+        notaccept: %r{(?:
+           an[ ]MX[ ]or[ ]SRV[ ]record[ ]indicated[ ]no[ ]SMTP[ ]service
+          |no[ ]host[ ]found[ ]for[ ]existing[ ]SMTP[ ]connection
+          )
+        }x,
+        # deliver.c:5614|  addr->message = US"delivery to file forbidden";
+        # deliver.c:5624|  addr->message = US"delivery to pipe forbidden";
+        # transports/pipe.c:1156|  addr->user_message = US"local delivery failed";
+        systemerror: %r{(?>
+           delivery[ ]to[ ](?:file|pipe)[ ]forbidden
+          |local[ ]delivery[ ]failed
+          |LMTP[ ]error[ ]after[ ]
+          )
+        }x,
+        # deliver.c:5425|  new->message = US"Too many \"Received\" headers - suspected mail loop";
+        contenterror: %r/Too[ ]many[ ]["]Received["][ ]headers/x,
+      }
+
+      # retry.c:902|  addr->message = (addr->message == NULL)? US"retry timeout exceeded" :
+      # deliver.c:7475|  "No action is required on your part. Delivery attempts will continue for\n"
+      # smtp.c:3508|  US"retry time not reached for any host after a long failure period" :
+      # smtp.c:3508|  US"all hosts have been failing for a long time and were last tried "
+      #                 "after this message arrived";
+      # deliver.c:7459|  print_address_error(addr, f, US"Delay reason: ");
+      # deliver.c:7586|  "Message %s has been frozen%s.\nThe sender is <%s>.\n", message_id,
+      # receive.c:4021|  moan_tell_someone(freeze_tell, NULL, US"Message frozen on arrival",
+      # receive.c:4022|  "Message %s was frozen on arrival by %s.\nThe sender is <%s>.\n",
+      ReDelayed = %r{(?:
+         retry[ ]timeout[ ]exceeded
+        |No[ ]action[ ]is[ ]required[ ]on[ ]your[ ]part
+        |retry[ ]time[ ]not[ ]reached[ ]for[ ]any[ ]host[ ]after[ ]a[ ]long[ ]failure[ ]period
+        |all[ ]hosts[ ]have[ ]been[ ]failing[ ]for[ ]a[ ]long[ ]time[ ]and[ ]were[ ]last[ ]tried
+        |Delay[ ]reason:[ ]
+        |Message[ ].+[ ](?:has[ ]been[ ]frozen|was[ ]frozen[ ]on[ ]arrival[ ]by[ ])
+        )
+      }x
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Exim'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->